src/software/networking/threaded_proto_multicast_sender.tpp

```diff
@@ -3,10 +3,12 @@
 template <class SendProto>
 ThreadedProtoMulticastSender<SendProto>::ThreadedProtoMulticastSender(
     const std::string& ip_address, const unsigned short port)
-    : io_service(), multicast_sender(io_service, ip_address, port)
+    : io_service(),
+      multicast_sender(io_service, ip_address, port),
+      io_service_thread([this]() { io_service.run(); })
 {
     // start the thread to run the io_service in the background
-    io_service_thread = std::thread([this]() { io_service.run(); });
+    //    io_service_thread = std::thread();
```

Comment:
Should this be commented out? If so, remove the inconsistent whitespace in the beginning.

Reply:
Yup, meant to remove it entirely. Removed now.